__tests__/app.test.js

```diff
@@ -193,4 +203,70 @@ describe("app()", () => {
         });
     });
   });
+
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("POST 201: /api/articles/:article_id/comments should return a 201 status code", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "I love this article",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then((data) => {
+          const { body } = data;
+          const comment = body.comments;
+          expect(comment).toMatchObject({
+            comment_id: 19,
+            body: "I love this article",
+            article_id: 3,
+            author: "butter_bridge",
+            votes: 0,
+            created_at: expect.any(String),
+          });
+        });
+    });
```

Comment:
Please write another test with status 201, where extra properties are ignored. For example where newComment looks like this:
` {
        username: "butter_bridge",
        body: "I love this article",
        extraProperty: "someValue"
      }`